unity/CITools/BuildDriver/Program.cs

```diff
@@ -48,8 +48,17 @@ public static async Task<int> Main(string[] args)
         Console.WriteLine($"\tConfiguration:\t\t{configuration}");
         Console.WriteLine("*****************************");
 
-        GlobalConfig gConfig = new (){ Architecture = architecture, Configuration = configuration, Silent = silent};
-        EmbeddingHost.Build(gConfig);
+        GlobalConfig gConfig = new ()
+        {
+            Architecture = architecture,
+            Configuration = configuration,
+            Silent = silent,
+            DotNetVerbosity = "quiet"
+        };
+
+        if (!skipBuild)
+            EmbeddingHost.Build(gConfig);
```

Comment:
The test workload was building this as well so I was ensuring this always built on purpose so it would stay in sync.

Reply:
I don't entirely follow.  I thought the test workflow didn't build anything?

What I observed was that both `dotnet build unity/managed.sln` and  `dotnet test unity/managed.sln` were being ran.  `dotnet test` will build so running `dotnet build` is unnecessary.

Reply:
https://github.com/Unity-Technologies/runtime/blob/2d8358c6899c8fb27283933d632cc6d76fa0315f/.yamato/scripts/test_windows.cmd#L37 -- Is the behavior I was attempting to preserve. The old test scripts were calling build then test.

Reply:
Turns out this was needed.  We need to build even when `skipBuild` is false because on CI we have the build and tests split into separate jobs.  And the way we have artifacts setup, we don't retain anything built under `unity`.  And therefore we need to rebuild it so that tests that depend on something in managed.sln can find what they need